internal/fsutil/create.go

```diff
@@ -9,14 +9,23 @@ import (
 )
 
 type CreateOptions struct {
-	Path string
-	Mode fs.FileMode
-	Data io.Reader
-	Link string
+	Path        string
+	Mode        fs.FileMode
+	Data        io.Reader
+	Link        string
+	MakeParents bool
 }
 
+// Creates file according to passed CreateOptions.
+// If o.MakeParents is true, missing parent directories of o.Path are
+// created with permissions 0755.
```

Comment:
Can we please move this comment to the options struct itself instead of here?

I've commented that earlier. Not sure why it wasn't part of the review.

Reply:
Moved the description of `MakeParents` to the structure field. Originally, there was a different comment at the structure field. I've added the current comments at the request of previous review.